RELEASE.md

```diff
@@ -80,61 +106,75 @@ Please make sure you have these tools installed before starting this process:
 
     Ensure the cherry-pick commit is free from any conflicts.
 
--   [ ] 8.2.  Push new Release Candidate tag
+    -   [ ] 7.2.  Push new Release Candidate tag
 
     Since there are new hotfix changes to the release, you must then add another release candidate tag to your release branch and push it again.
 
     ```bash
     git tag '6.0.0-RC-<n>' && git push --tag
     ```
 
--   [ ] 9.  Test
+## 3. Testing and stabilizing the release
```

Comment:
This section on testing still needs some work. It seems that some parts of this testing process are currently repeated?

We need to clarify better what are the responsibilities of the following roles:

-   Release Manager
-   QA team
-   Solutions Engineers
-   Any other stakeholders for the release

This is just a suggestion, but it could also be a good idea to involve the Product Managers who were involved in the changes to the product at some point during this stage of the release process.

Reply:
I agree that the steps could be better distributed by the different roles. Why don't we try to do this already?
We could nest everything like:
**Approval by Release Manager** steps: 5
**Approval by QA** steps: 1, 2, 3, 4, 6, 9, 10
**Approval by Solutions Engineers** steps: 8, 10

Reply:
I did a big overhaul of this entire section, separating the tasks by the responsibilities of each stakeholder.

I believe the tasks that must be done are now much easier to understand, but please check if something is out of place.

In particular, I grouped the Engineering teams and the QA team under the same section as I believe there are synergies to take advantage of while doing the exploratory tests and also while debugging the failed regression tests.